hloc/extract_features.py

```diff
@@ -14,6 +14,17 @@
 from .utils.tools import map_tensor
 
 
+import torch.utils.data
+from PIL import Image
+
+import torchvision
+normalize = torchvision.transforms.Normalize(mean=[0.485, 0.456, 0.406],
+                                 std=[0.229, 0.224, 0.225])
+transform = torchvision.transforms.Compose([
+                                       torchvision.transforms.ToTensor(),
+                                       normalize])
```

Comment:
I also tried this one, but somehow when `transform` is called within the new `DIR`class it can't process the data directly as a tensor; it requires a PIL format. On the other hand, this transformation is standard for a majority of PyTorch models so maybe having it within `extract_features.py` would be helpful when adding other networks for feature extraction.